pkg/gadgets/top/file/types/types.go

```diff
@@ -43,18 +36,30 @@ const (
 type Stats struct {
 	eventtypes.CommonData
 
+	Pid        uint32 `json:"pid,omitempty" column:"pid,template:pid"`
+	Tid        uint32 `json:"tid,omitempty" column:"tid,template:pid,hide"`
+	Comm       string `json:"comm,omitempty" column:"comm,template:comm"`
 	Reads      uint64 `json:"reads,omitempty" column:"reads"`
 	Writes     uint64 `json:"writes,omitempty" column:"writes"`
 	ReadBytes  uint64 `json:"rbytes,omitempty" column:"rbytes"`
 	WriteBytes uint64 `json:"wbytes,omitempty" column:"wbytes"`
-	Pid        uint32 `json:"pid,omitempty" column:"pid"`
-	Tid        uint32 `json:"tid,omitempty" column:"tid"`
-	MountNsID  uint64 `json:"mountnsid,omitempty" column:"mountnsid"`
-	Filename   string `json:"filename,omitempty" column:"filename"`
-	Comm       string `json:"comm,omitempty" column:"comm"`
-	FileType   byte   `json:"fileType,omitempty" column:"fileType"` // R = Regular File, S = Socket, O = Other
+	MountNsID  uint64 `json:"mountnsid,omitempty" column:"mountnsid,template:ns,hide"`
+	FileType   byte   `json:"fileType,omitempty" column:"T,maxWidth:1"` // R = Regular File, S = Socket, O = Other
+	Filename   string `json:"filename,omitempty" column:"file"`
 }
 
 func GetColumns() *columns.Columns[Stats] {
-	return columns.MustCreateColumns[Stats]()
+	cols := columns.MustCreateColumns[Stats]()
+
+	cols.MustSetExtractor("rbytes", func(stats *Stats) (ret string) {
```

Comment:
`rsize`? (Given that it's not only bytes)

Reply:
I see the reasoning behind that.
Would we want to keep the json field in sync? Or leave it at `rbytes`. The membervariable should be renamed then, too

Reply:
I think you're right, we should change field and json tag name too. I think this is a longer discussion, I'll leave for another PR not to delay this one.